.github/workflows/post-merge.yaml

```diff
@@ -55,44 +55,3 @@ jobs:
         # Only run if required secrets are provided
         if: ${{ env.DOCKER_USER && env.DOCKER_PASSWORD }}
         run: make deploy-ci
```

Comment:
Any reason for removing the e2e on master ? Even though we run it on a PR, it would fine to run it on master as well imo.

Reply:
Hmm good point. We can have both. I'll restore them.

Reply:
Updated, it's running twice now. Slightly refactored the actions in the process.